### ISSUE TITLE
A11Y: Improve aria-label for Overview link

### ISSUE THREAD

**Issue by steinsebastian** [SAP/spartacus#15675]
For blind users the overview page has to be accessed via read-mode of screen readers. 
That should be announced on the active "Overview" link on the overview page.

Currently it reads: "You are on the overview page".

Discussed with product owner: 
It shall read: "You are on the overview page. Check attributes and their selected values with read-mode of your screen-reader".
